user/languages/rust.md

```diff
@@ -86,6 +86,19 @@ cache: cargo
 ```
 {: data-file=".travis.yml"}
 
+This adds the following directories to the cache:
+
+- `$TRAVIS_HOME/.cache/sccache`
+- `$TRAVIS_HOME/.cargo/`
+- `$TRAVIS_HOME/.rustup/`
+- `target`
+
+In addition, it adds the following command to the `before_cache`
+phase of the job in order to reduce cache size:
+
+    rm -rf "$TRAVIS_HOME/.cargo/registry/src"
```

Comment:
```suggestion
    `rm -rf "$TRAVIS_HOME/.cargo/registry/src"`
```

Reply:
This is not necessary because with 4 leading white spaces Markdown will style this in a monospace font.